packages/teleterm/src/ui/QuickInput/useQuickInput.ts

```diff
@@ -136,18 +160,18 @@ export default function useQuickInput() {
   // Reset active suggestion when the suggestion list changes.
   // We extract just the tokens and stringify the list to avoid stringifying big objects.
   // See https://github.com/facebook/react/issues/14476#issuecomment-471199055
+  // TODO: It's another action we should perform after the input has changed.
```

Comment:
This one seems tricky because the state is held in the `useAsync` hook. The second link in this todo has a great example of what we want, but not quite the shape of our component. I'll update the TODO as you suggested.